### PR TITLE
5075-upgrade braintree/sanitize-url

### LINKED ISSUE #5075: [SNYK: Medium] @braintree/sanitize-url Cross-site Scripting (XSS) (Due 05/09/2022)
**Overview**
@braintree/sanitize-url is an A url sanitizer

**Detailed paths and remediation**
Introduced through: openfec@1.0.0 › @braintree/sanitize-url@2.1.0
Fix: [Upgrade](https://app.snyk.io/org/fecgov/fix/d0baf872-87d8-4d68-80a5-cb52a6d0f57b?vuln=SNYK-JS-BRAINTREESANITIZEURL-2339882) to [@braintree/sanitize-url](https://www.npmjs.org/package/@braintree/sanitize-url)@6.0.0 

Affected versions of this package are vulnerable to Cross-site Scripting (XSS) due to improper sanitization in sanitizeUrl function.

https://security.snyk.io/vuln/SNYK-JS-BRAINTREESANITIZEURL-2339882

### PR DESCRIPTION
**Pull request by cnlucas**
## Summary

- Resolves #5075 

Upgrades braintree/sanitize-url to v.6.0.0 remediate vulnerabilities to Cross-site Scripting (XSS) due to improper sanitization in sanitizeUrl function.

### Required reviewers
1-2 devs 

## Impacted areas of the application

General components of the application that this PR will affect:

-  openFEC

## Screenshots
Before:
Screen Shot 2022-03-16 at 5 10 08 PM
After:
Screen Shot 2022-03-16 at 5 03 50 PM

## How to test
- run `snyk test` locally on develop to confirm vulnerability
`snyk test --file=requirements.txt --package-manager=pip`
`snyk test --file=package.json`
- checkout this branch
-  `rm -rf node_modules`
- `pip install -r requirements.txt`
- `pip install -r requirements-dev.txt`
- `npm install && npm run build`
- run `snyk test` 
you should see one less vulnerability (see screenshots above)

Also--- thank you @pkfec and @fec-jli (I got most of these snyk instructions from you both!)

### CODE REVIEW

**Comment by pkfec** [comment]
@cnlucas Just an fyi, Maven plugin is required to check for vulnerabilities on *.jar files. You can run the following snyk commands to check for vulnerabilities on `requirements*.txt` and `package*.json files`.

```
snyk test --file=requirements.txt --package-manager=pip

snyk test --file=package.json
```

**Comment by pkfec** [comment]
You need to add one little thing to `How to test` instructions, before installing the dependencies:

First : rm -rf node_modules 
Later: npm i && npm run build

**Comment by cnlucas** [comment]
Thanks @pkfec, your edits are added. Good to know about the specific snyk commands!

**Review by fec-jli — APPROVED** [review]
Looks good. Thanks

File: package-lock.json [hunk 1]
```diff
@@ -89,8 +89,8 @@
     },
     "node_modules/@braintree/sanitize-url": {
       "version": "2.1.0",
```
Comment (pkfec) [inline package-lock.json:91]: Version is still referring to 2.1.0. It should be 6.0.0

Comment (cnlucas) [reply package-lock.json]: Thanks for catching this, I've made the change, my apologies for the oversight! @pkfec

Comment (pkfec) [reply package-lock.json]: @cnlucas Looks good!

**Pull request merged**